packages/amplication-client/src/Workspaces/hooks/useProjectSelector.ts

```diff
@@ -71,7 +71,11 @@ const useProjectSelector = (
   useEffect(() => {
     if (loadingList || !projectListData) return;
 
-    setProjectList(projectListData.projects);
+    const sortedProjects = [...projectListData.projects].sort((a, b) => {
```

Comment:
I think we should use server-side sorting

Reply:
What if in the future we want to offer different sorting filters? IMHO server response should contain enough details to allow client the sorting they prefer. 
If we want to enforce a default sorting, we should consider an order/index property instead of relying on the intrinsic order of the Array in the json response.

Reply:
The graphQL API supports a parameter for sorting that also takes pagination into consideration. 
Since the list of projects is usually short and we anyway don't use pagination now, we can use client-side sorting - but we already have support for server-side sorting, so all we need to do is to send the name of the field to the server

![image](https://user-images.githubusercontent.com/43705455/204250516-6b4d0050-38b3-4ad5-874e-3b22a65b349e.png)